kustomize/base/brigadasos-db.yaml

```diff
@@ -24,12 +27,45 @@ metadata:
     app: brigadasos-db
 spec:
   storageClassName: manual
+  volumeName: brigadasos-db-pvolume
   accessModes:
     - ReadWriteMany
   resources:
     requests:
       storage: 50Gi
 ---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: brigadasos-db-temp-pvolume
+  namespace: nadedb
+  labels:
+    type: local
+    app: brigadasos-db
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 50Gi
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/mnt/nadedb"
+---
```

Comment:
Related to the comment below, this new  `PresistentVolume` for `temp` is also missing the `claimRef` attribute as you specified above for the other `PersistentVolume`